cea/utilities/reporting.py

```diff
@@ -40,18 +38,25 @@ def full_report_to_xls(tsd, output_folder, basename):
     writer.close()
 
     # quick visualization
-    quick_visualization_tsd(tsd, output_folder, basename)
+    # UNCOMMENT THE NEXT LINE TO PRODUCE QUICK VISUALIZATIONS WITH PLOTLY OF THE DEMAND CALCULATION IN BUILDINGS
+    # quick_visualization_tsd(tsd, output_folder, basename)
 
 
 def quick_visualization_tsd(tsd, output_folder, building_name):
 
-    plot_heat_load = False
-    plot_heat_temp = False
+    # import keys
+    from cea.demand.thermal_loads import TSD_KEYS_HEATING_LOADS, TSD_KEYS_HEATING_TEMP, TSD_KEYS_RC_TEMP, \
+        TSD_KEYS_COOLING_LOADS, TSD_KEYS_MOISTURE, TSD_KEYS_VENTILATION_FLOWS, TSD_KEYS_COOLING_SUPPLY_TEMP, \
+        TSD_KEYS_COOLING_SUPPLY_FLOWS
+
+    # set to True to produce plotly graphs of selected variables
```

Comment:
again, this code assumes the user will change the _source_ to turn functionality on or off. let's not do that. this is what the config file is for. if you don't want to create a whole bunch of options, just have one option (quick-visualization-tsd) or something. then, it is documented and further users / researchers will learn about the functionality from the user interface.

Reply:
See above.